subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -737,36 +750,40 @@ static void rp_comm_tx(struct ll_conn *conn, struct proc_ctx *ctx)
 		ctx->rx_opcode = PDU_DATA_LLCTRL_TYPE_LENGTH_RSP;
 		break;
 #endif /* CONFIG_BT_CTLR_DATA_LENGTH */
-#if defined(CONFIG_BT_CTLR_DF_CONN_CTE_REQ)
+#if defined(CONFIG_BT_CTLR_DF_CONN_CTE_RSP)
 	case PROC_CTE_REQ: {
 		uint8_t err_code = 0;
 
 		if (conn->llcp.cte_rsp.is_enabled == 0) {
 			err_code = BT_HCI_ERR_UNSUPP_LL_PARAM_VAL;
 		}
+
 #if defined(CONFIG_BT_PHY_UPDATE)
 		/* If the PHY update is not possible, then PHY1M is used.
 		 * CTE is supported for PHY1M.
 		 */
-		if (conn->lll.phy_tx != PHY_CODED) {
+		if (conn->lll.phy_tx == PHY_CODED) {
```

Comment:
Shouldn't this be 
```suggestion
		if (conn->lll.phy_tx != PHY_1M {
```
based on the comment?

Reply:
The comment was wrong. It should say that the CTE is not supported with CODED PHY.